cobbler/web/urls.py

```diff
@@ -6,7 +6,8 @@
 # from cobbler_web.contrib import admin
 # admin.autodiscover()
 
-urlpatterns = [
+urlpatterns = patterns(
```

Comment:
I still don't get why this change ? Any django urls.py created after django-admin startproject foo with not have this patterns('', ...)

Reply:
I can't recall for sure, but this was probably the result of trial-and-error. I don't claim to know much about django (or cobbler...)

Reply:
Yep, this is wrong. The correct way has already been committed.